Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/qrcode/CoronaTestQRCode.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.coronatest.qrcode
+
+import android.os.Parcelable
+import de.rki.coronawarnapp.coronatest.type.CoronaTest
+import kotlinx.parcelize.Parcelize
+import org.joda.time.Instant
+
+sealed class CoronaTestQRCode : Parcelable {
```

Comment:
does it have to be Parcelable ?

Reply:
Not guaranteed in the final version, but for the purpose of us parallelising work, it should be for now. Navigation is not fully clear yet, and when switching to different screens @axelherbstreith has the option of passing the scanned QR Code along and completing the flow without requiring to scan again, or to persist the data to disk, giving him a lot of flexibility when working on the UI flow.